src/Components/InputBook.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Great job working on this milestone. Eslint are there to make your code robust. Kindly fix the errors raised rather than disabling the eslint